web-app/django/VIM/apps/instruments/templates/instruments/index.html

```diff
@@ -93,10 +94,9 @@ <h4 class="ms-3 me-2 my-auto"><small>INSTRUMENT LIST</small></h4>
       </div>
 
       <div class="d-flex">
-        <button type="button" class="btn me-1 px-1 py-0 justify-content-center display-btn" id="page-setting-btn" data-bs-toggle="modal" data-bs-target="#paginationModal">
+        {% comment %} <button type="button" class="btn me-1 px-1 py-0 justify-content-center display-btn" id="page-setting-btn" data-bs-toggle="modal" data-bs-target="#paginationModal">
```

Comment:
Do you want to keep this comment in?

Reply:
It seems that we don't need the setting button at present, I'll delete it.